src/emqx.appup.src

```diff
@@ -1,6 +1,9 @@
 %% -*- mode: erlang -*-
 {VSN,
  [
+   {"4.3.3", [
+     {load_module, emqx_channel, brutal_purge, soft_purge, []}
```

Comment:
4.3.3 is the upcoming release version, so it is from 4.3.2, 4.3.1, 4.3.0 upgrade and downgrade 4.3.3, so you need to add load_module to the corresponding version

Reply:
Thanks! Sorry about this. I've managed to avoid appup files for 20 years, so I don't know how they are used ;-)